docs/UserGuide.md

```diff
@@ -91,7 +91,20 @@ Examples:
 
 Shows a list of all persons in the address book.
 
-Format: `list`
+Format: `list [s/FIELD]`
+
+* Sorts the contacts by the specified field in **ascending** order. `FIELD` must take one the following values:
+  * `n` sort by name
+  * `d` Sort by date of birth
+  * `g` Sort by gender
+
+* It is optional to specify the field to sort by. If no field is specified, persons are listed in the order they were inserted.
+* At most one field can be specified. i.e. Cannot specify 2nd or 3rd criteria to sort by.
+* Persons with an empty field that is being used to sort will be placed at the top of the list.
+
+Examples:
+* `list` Lists all persons without sorting them.
+* `list s/n` Lists all persons sorted by their names.
```

Comment:
On a side note, I think it might be helpful to have an example for each variation of the parameter though its not compulsory!
* `list s/d` Lists all persons sorted by their date of birth.
* `list s/g` Lists all persons sorted by their gender.

Reply:
Another suggestion would be to reiterate the order of sorting under examples again
* `list s/n` Lists all persons sorted by their names in ascending order.